sydney/sydney.py

```diff
@@ -91,10 +95,31 @@ def _build_ask_arguments(self, prompt: str) -> dict:
             "responsible_ai_policy_235",
             "enablemm",
             "dv3sugg",
+            "iyxapbing",
+            "iycapbing",
+            "galileo",
+            "saharagenconv5",
+            "h3imagntvwcp",
+            "seqnumtts",
+            "log2sph",
+            "savememfilter",
+            "uprofgen",
+            "uprofupd",
+            "uprofupdasy",
+            "eredirecturl",
```

Comment:
Here was what was missing.
I'm not sure which option does it, or what any of these even do (internal names for features?), maybe you know what they do. But adding the same ones that were being added on the web interface fixed it, and it can now see the images you upload.

Reply:
Awesome catch! Very often these flags come and go and it can be hard to keep track of them. Sometimes features work without all of them, sometimes they don't. Unfortunately, I am not familiar with several of them, but in the future I'd definitely like to be able to properly document those.